Gemfile

```diff
@@ -32,8 +33,7 @@ gem 'memo_wise'
 gem 'nokogiri'
 gem 'omniauth-google-oauth2'
 gem 'omniauth-rails_csrf_protection'
-# Source from RubyGems after https://github.com/paper-trail-gem/paper_trail/pull/1511 is released.
-gem 'paper_trail',
+gem 'paper_trail', # Source from RubyGems after https://github.com/paper-trail-gem/paper_trail/pull/1511 is released.
```

Comment:
Having a comment above `gem 'paper_trail'` interferes with RuboCop's reordering of gems. By having just a list of `gem`s without any interruptions, RuboCop can reorder them properly.